tests/conftest.py

```diff
@@ -60,11 +60,18 @@ def __init__(self, testbed_file):
                 del line['topo']
                 line['topo'] = defaultdict()
                 line['topo']['name'] = topo
+                line['topo']['type'] = self.get_testbed_type(line['topo']['name'])
                 with open("../ansible/vars/topo_{}.yml".format(topo), 'r') as fh:
                     line['topo']['properties'] = yaml.safe_load(fh)
 
                 self.testbed_topo[line['conf-name']] = line
 
+    def get_testbed_type(self, topo_name):
+        testbed_type = ['t0', 't1', 'ptf']
+        for val in testbed_type:
+            if topo_name.find(val) != -1:
```

Comment:
suggest to use regex match somehting like ("^(t0|t1|ptf)[-]")?

Reply:
@lguohan , thank you for your advice. I used regex match() to check if topo_name matches expected topo_type. Could you please review it?